app/migrations/20241003214418-create_assistant_messages_table.cjs

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const sql_up = `create type role_enum as enum('user', 'assistant');
+
+create table if not exists public."AssistantMessage"
+(
+    assistant_message_id  text not null primary key,
+    thread_id             text not null,
```

Comment:
Can you use UUIDs (v4) for the table IDs? All our other tables do this, so it's preferable to keep this pattern.
Comparing and indexing UUID columns is more efficient compared to using arbitrary strings.

Reply:
the ids come from openAPI

Reply:
Yes the IDs are in the form of 'thread_deMv.......' therefore we are not using the UUID here